src/golang/jobs/test-nodb.yml

```diff
@@ -45,11 +53,65 @@ steps:
       condition: << parameters.checkout >>
       steps:
         - checkout
+  - when:
+      condition: << parameters.cache >>
+      steps:
+        - restore_cache:
+            keys:
+              - v1-mod-cache-{{ checksum "go.sum" }}
+              - v1-mod-cache-
+  - when:
+      condition: << parameters.check-modules >>
+      steps:
+        - go:
+            pkg-spec: ""
+            opts: mod verify
+        - go:
+            pkg-spec: ""
+            opts: mod tidy
+        - run:
+            name: Check Modules
+            command: |
+              if [ ! -z "$(git status --porcelain)" ]; then
+                echo
+                echo -e '\033[1;93m!!! "go mod verify && go mod tidy" resulted in changes. Please run those commands locally and commit the changes.\e[0m'
```

Comment:
It doesn't look like the error you linked actually does anything with the formatting codes. Were you seeing something different? If not maybe they can be removed.

Reply:
it highlight the message as yellow for me.

Reply:
![Screen Shot 2019-10-24 at 6 51 31 PM](https://user-images.githubusercontent.com/56555229/67537282-86924580-f68f-11e9-91bc-e8a5faad0359.png)

Reply:
oh, I think I was just looking at the output on circle-ci, I guess if it is run locally it would highlight. :+1: